src/ui/components/DevToolsProcessingScreen.tsx

```diff
@@ -30,8 +53,21 @@ export function DevToolsProcessingScreen() {
     }
   }, [processingProgress]);
 
+  useEffect(() => {
+    return () => {
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      const { permission, permissionsRequested } = notificationPermissionStateRef.current;
+      if (permissionsRequested && permission === "granted" && recording) {
+        const progress = getProcessingProgress(store.getState());
+        if (progress === 100) {
+          new Notification(`"${recording.title}" has loaded`);
```

Comment:
we could add a `tag` option to avoid creating multiple notifications when the user would have that recording open multiple times in their tabs. The tag could contain the recording's ID and the status so notification related to different recordings wouldn't cancel each other

Reply:
Great suggestion! I was not aware of the tag API.